extras/mainnet.json

```diff
@@ -66,6 +66,7 @@
     },
     "helpers": {
         "balance_checker": "0xe92261c2D64C363109c36a754A87107142e61b72",
+        "vlaura_relocker_module": "0xfD20C63554A9916816dC5e5Df596A0333185F263",
```

Comment:
The deployer here is not listed anywhere in our address book.  Who was the deployer here.  Has someone checked that the code matches exactly the reviewed contract?

Reply:
Have a link to the code/review?  Can we add deployer to our addressbook?